x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/indexlifecycle/OperationModeUpdateTask.java

```diff
@@ -27,6 +28,13 @@ OperationMode getOperationMode() {
 
     @Override
     public ClusterState execute(ClusterState currentState) {
+        ClusterState newState = currentState;
+        newState = updateILMState(newState);
+        newState = updateSLMState(newState);
```

Comment:
It's a little weird that SLM only transitions from `STOPPING` to `STOPPED` once ILM decides it's safe to stop, despite technically being separate. This is probably fine, especially for now, just surprised me a bit.